cypress/integration/1217-my_profile_page/352-user_opens_the_page_my_profile/open_my_profile.js

```diff
@@ -12,18 +12,8 @@ When('they click the menu item My Profile', () => {
 });
 Then('they see My Profile', () => {});
 And('they see their Profile page which includes information for', (dataTable) => {
-  const fieldMap = {
-    'Real name': 'user-name',
-    'Feide ID': 'user-id',
-    Email: 'user-id',
-    ORCID: 'button-create-connect-orcid',
-    'Role(s)': 'user-role-creator',
-    Institution: 'institution-presentation',
-    'Preferred language': 'language-selector',
-  };
-  dataTable.rawTable.forEach((value) => {
-    cy.get(`[data-testid=${fieldMap[value[0]]}]`).should('exist');
-  });
+  const fieldMap = PROFILE_PAGE_FIELDS;
+  cy.testDataTestidList(dataTable, fieldMap);

```

Comment:
Trenger du egen const for fieldMap? ser ut som det alltid bare sendes videre (gjelder alle andre plasser også)
```suggestion
  cy.testDataTestidList(dataTable, PROFILE_PAGE_FIELDS);
```

Reply:
poeng...